src/components/Dashboard/DashboardItem/ItemBase.js

```diff
@@ -0,0 +1,308 @@
+            underline="always"
+        >
+            {children}
+        </Link>
+    ) : (
+        <Link href={target} color="inherit" underline="always">
+            {children}
+        </Link>
+    );
+};
+
+/**
+ * An item in the dashboard.
+ *
+ * @param {Object} props - The props for the component.
+ * @param {String} props.kind - The kind of item. Example: "app" or "analysis".
+ * @param {Object} props.content - The content for the item returned from the API.
+ * @returns {Object}
+ */
+const DashboardItem = ({ item }) => {
+    const theme = useTheme();
+    const color = getSectionColor(item.section, theme);
+    const classes = useStyles({
+        width: item.width,
+        height: item.height,
+        color,
+    });
+
+    const { t } = useTranslation("dashboard");
+
+    const isMediumOrLarger = useMediaQuery(theme.breakpoints.up("md"));
+
+    const cardID = item.id;
+    const user = item.username;
+
+    const description = fns.cleanDescription(item.content.description);
+    const [origination, date] = item.getOrigination(t);
+
+    const [menuAnchorEl, setMenuAnchorEl] = useState(null);
+    const menuID = fns.makeID(cardID, "menu");
+
+    return (
+        <Card
+            classes={{ root: classes.dashboardCard }}
+            id={fns.makeID(ids.ITEM, cardID)}
+            elevation={4}
+        >
+            <CardHeader
+                avatar={
+                    isMediumOrLarger && (
+                        <Avatar className={classes.avatar}>
+                            {item.getAvatarIcon(classes)}
+                        </Avatar>
+                    )
+                }
+                classes={{
+                    root: classes.cardHeaderDefault,
+                    content: classes.cardHeaderContent,
+                }}
+                title={item.content.name}
+                titleTypographyProps={{
+                    noWrap: true,
+                    variant: "subtitle2",
+                    classes: { colorPrimary: classes.cardHeaderText },
+                }}
+                subheader={`${origination} ${user} on ${date}`}
+                subheaderTypographyProps={{
+                    noWrap: true,
+                    variant: "caption",
+                    classes: { colorPrimary: classes.cardHeaderText },
+                }}
+            />
+            <CardContent
+                classes={{
+                    root: classes.root,
+                }}
+            >
+                <Typography color="textSecondary" variant="body2" component="p">
+                    {description || t("noDescriptionProvided")}
+                </Typography>
+            </CardContent>
+
+            <CardActions disableSpacing>
+                {item.actions}
+                {item.menuActions.length > 0 && (
+                    <>
+                        <IconButton
+                            className={classes.tripleDotMenu}
+                            aria-controls="action-menu"
```

Comment:
If this button is responsible for opening the menu below, then `aria-controls` needs to have the element ID for the menu.  In this case, `menuID` looks like.